src/ensembl/src/content/app/entity-viewer/components/example-links/ExampleLinks.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { useQuery } from '@apollo/react-hooks';
+import { gql } from 'apollo-boost';
+
+import * as urlHelper from 'src/shared/helpers/urlHelper';
+import { buildFocusIdForUrl } from 'src/shared/state/ens-object/ensObjectHelpers';
+
+import { getEntityViewerActiveGenomeId } from 'src/content/app/entity-viewer/state/general/entityViewerGeneralSelectors';
+import { getGenomeExampleFocusObjects } from 'src/shared/state/genome/genomeSelectors';
+
+import { RootState } from 'src/store';
+import { ExampleFocusObject } from 'src/shared/state/genome/genomeTypes';
+
+import styles from './ExampleLinks.scss';
+
+type ExampleGene = {
+  id: string;
+  symbol: string;
+};
+
+type ExampleLinksProps = {
+  activeGenomeId: string | null;
+  exampleEntities: ExampleFocusObject[]
+};
+
+const QUERY = gql`
+  query Gene($id: String!) {
+    gene(byId: { id: $id }) {
+      id
+      symbol
+    }
+  }`;
+
+// NOTE: the component currently handles only example gene
+const ExampleLinks = (props: ExampleLinksProps) => {
+  const exampleGeneId = props.exampleEntities.find(({ type }) => type === 'gene')?.id;
+  const { loading, data } = useQuery<{ gene: ExampleGene }>(QUERY, {
```

Comment:
`loading` is unused.

Reply:
Added a spinner to show for the `loading` state.